Dockerfile

```diff
@@ -65,6 +65,35 @@ COPY --from=build-src /usr/share/licenses/testsys /licenses/testsys
 
 ENTRYPOINT ["./ec2-resource-agent"]
 
+# =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^=
+# Builds the EC2 karpenter resource agent image
+FROM public.ecr.aws/amazonlinux/amazonlinux:2 as ec2-karpenter-resource-agent
+
+RUN yum install -y git make tar \
+    && curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 \
+    && chmod +x get_helm.sh && ./get_helm.sh --version v3.8.2
+# Copy eksctl
```

Comment:
We should check a sha, or create an issue where we need to check shas for all of the things we download from various places in this dockerfile.

```suggestion
RUN yum install -y git make tar \
    && curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 \
    && chmod +x get_helm.sh && ./get_helm.sh --version v3.8.2

# Copy eksctl
```

Reply:
I am working on fixing that.